### PR TITLE
Security vulnerabilities

### PR DESCRIPTION
**Pull request by marcelotoledo5000**
# Because

## We found potential security vulnerabilities in your dependencies.

Update versions to fix it.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+* Improvements
+
+  *Marcelo Toledo*
+
+## Gull 1.0.1 (Aug 3, 2018)
+
+* Bump version
+
+  *Marcelo Toledo*
```
Comment (jao) [inline CHANGELOG.md]: Tá meio errado esse change log...

**Comment by jao** [comment]
Falhou o teste...

**Comment by gchapim** [comment]
Esse fix não vai resolver o problema do omniauth

**Comment by gchapim** [comment]
Eu resolvi no koala a falha de segurança do omniauth assim: https://github.com/petlove/koala/issues/782 (tive que trocar a gem)

File: CHANGELOG.md (hunk 1 above)
Comment (marcelotoledo5000) [reply CHANGELOG.md]: Ahahahahahah, famoso `copy` and `paste`... 

Arrumei...

**Pull request merged**